src/app.test.ts

```diff
@@ -9,3 +10,39 @@ describe('GET /', () => {
       .expect(200, done);
   });
 });
+
+describe('GET /flights', () => {
+  const badParamsCode = 400;
+  it('returns an error when date is not provided ', (done) => {
+    supertest(app)
+      .get('/flights')
+      .expect(badParamsCode, done);
+  });
+
+  it('returns an error when date is malformed ', (done) => {
+    supertest(app)
+      .get('/flights?date=junk')
+      .expect(badParamsCode, done);
+  });
+
+  it('retrieves the same data when hit multiple times, the second call should hit the cache ', (done) => {
```

Comment:
Since this test isn't actually testing the cache yet, lets remove the part after comma for now